src/__test__/iotx.integration.test.ts

```diff
@@ -47,6 +47,17 @@ test.serial("transfer throws if no account", async t => {
   );
 });
 
+test.serial("change provider", async t => {
+  // tslint:disable-next-line:no-http-string
+  const antenna = new Antenna("http://api.testnet.iotex.one:80");
+  let chainMeta = await antenna.iotx.getChainMeta({});
+  t.truthy(chainMeta);
+  // tslint:disable-next-line:no-http-string
+  antenna.setProvider("http://api.iotex.one:80");
```

Comment:
let's use the interface like this:
```
setProvider(provider: string | IRpcMethod): void;
```

and also expose currentProvider in antenna